utils-kotlin/src/test/kotlin/org/sonarsource/kotlin/externalreport/common/TranslatorTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.sonarsource.kotlin.externalreport.common
```

Comment:
Here as well, we should put a license header. I realize that we haven't done so for the other tests in this module, something to update at some point too.